modules/xfem/src/userobjects/EllipseCutUserObject.C

```diff
@@ -91,3 +91,18 @@ EllipseCutUserObject::isInsideCutPlane(Point p) const
   }
   return false;
 }
+
+void
+EllipseCutUserObject::calculateCrackFrontPoints()
+{
+  Point v1 = _unit_vec1;
+  Point v2 = _unit_vec2;
+  _crack_front_points.resize(_number_crack_front_points);
+  // parametric circle in 3D: center + r * cos(theta) * v1 + r * sin(theta) * v2
+  for (unsigned int i = 0; i < _number_crack_front_points; ++i)
+  {
+    Real theta = 2.0 * libMesh::pi / _number_crack_front_points * i;
+    _crack_front_points[i] =
+        _center + _long_axis * std::cos(theta) * v1 + _short_axis * std::sin(theta) * v2;
+  }
+}
```

Comment:
It would be nice to have these uniformly spaced by arc length rather than by the angle on the parametric circle before we merge this, although it looks like this is a little bit messy. Looks like you need to use an elliptic integral. 
http://mathworld.wolfram.com/Ellipse.html
https://math.stackexchange.com/questions/172766/calculating-equidistant-points-around-an-ellipse-arc
There are a number of approximations of elliptic integrals out there that aren't too bad to implement.

To avoid holding this commit up, maybe we should just skip the ellipse for now while we sort that out.